src/views.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import APIRouter, HTTPException
+from src.managers import DepartmentManager
+
+router = APIRouter(prefix="/departments")
+
+
+@router.post("/")
+def create_item(department: dict):
+    DepartmentManager.insert_data(data=department)
+    return {"Data added successfully"}
+
+
+@router.get("/", name="get_departments")
+def read_depart():
+    department = DepartmentManager.get_info()
+    return department
```

Comment:
Let's refactor a bit
```suggestion
    return DepartmentManager.get_info()
```